recipe/entry_point.py

```diff
@@ -56,7 +63,15 @@ def map(self, func, *iterables):
            import tqdm
            from conda_package_handling import api
            from concurrent.futures import ProcessPoolExecutor
-           executor = ProcessPoolExecutor()
+
+           if args.num_processors:
+               if args.num_processors < 1:
+                   raise ValueError("--num-processors must be an integer greater than or equal to 1")
+               if os.name == "nt":
+                   # See Windows notes: https://docs.python.org/3/library/concurrent.futures.html#processpoolexecutor
+                   args.num_processors = min(args.num_processors, 61)
+
+           executor = ProcessPoolExecutor(max_workers=args.num_processors)
```

Comment:
Seeing this now, is `conda_package_handling` thread/processor safe? I think I heard it wasn't... @dholth do you have any insights?

Reply:
2.0 will be. Should have already been process safe. https://dholth.github.io/conda-benchmarks/#extract.TimeExtract.time_extract?conda-package-handling=2.0.0a2&p-format='.conda'&p-format='.tar.bz2'&p-lang='py'&x-axis=format shows that 3 is the correct number of threads to use.

Reply:
@dholth Thanks, this is good info. I've updated the default value from `None` (i.e. max possible) to `max(3, os.cpu_count())`.